Formula/ices0.rb

```diff
@@ -0,0 +1,47 @@
+class Ices0 < Formula
+  desc "Source client for broadcasting to Icecast/Shoutcast servers in MP3 format"
+  homepage "https://github.com/Moonbase59/ices0"
+  url "https://github.com/Moonbase59/ices0/archive/v0.4.11.tar.gz"
+  sha256 "9179496149e763a75fea37bf6ec12947cad4bdb868b401d34728b353b836047a"
+  head "https://github.com/Moonbase59/ices0.git"
+
+  # building
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  # input/output
```

Comment:
Are all of these runtime dependencies?

Reply:
I don’t know if I only get shown part of it, but libtool and pkg-config are just build deps (as denoted by `=> :build`.